seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/task/flow/SinkFlowLifeCycle.java

```diff
@@ -97,6 +103,8 @@ public class SinkFlowLifeCycle<T, CommitInfoT extends Serializable, AggregatedCo
 
     private EventListener eventListener;
 
+    private final List<Map<TablePath, TablePath>> tablesMaps = new ArrayList<>();
```

Comment:
share more details about tablesMaps on comment? Why we need this? And how to use it.